Headers/AppKit/NSImageRep.h

```diff
@@ -177,6 +177,15 @@ enum {
 
 @end
 
+typedef struct CGImage *CGImageRef;
+@interface NSImageRep (GSQuartz)
+#if OS_API_VERSION(MAC_OS_X_VERSION_10_6, GS_API_LATEST)
+- (CGImageRef)CGImageForProposedRect: (NSRect *)proposedDestRect 
+                         context: (NSGraphicsContext *)referenceContext 
```

Comment:
`context:` and `hints:`'s colons are not aligned well below `CGImageForProposedRect:`'s colon. Please fix and I will merge.